targets/wasi.json

```diff
@@ -7,6 +7,7 @@
 	"goarch":        "arm",
 	"linker":        "wasm-ld",
 	"libc":          "wasi-libc",
+	"rtlib":         "compiler-rt",
```

Comment:
Actually now I realize this probably needs to be added to targets/wasm.json too. Feel free to make a new PR if you'd like.

Reply:
New PR for `wasm.json` https://github.com/tinygo-org/tinygo/pull/3848